package.json

```diff
@@ -42,31 +42,37 @@
   },
```

Comment:
All you need to update in this file in order to get rid of `hoek` vulnerability is in this patch:
```
diff --git a/package.json b/package.json
index 83ebba7..baf4ac1 100644
--- a/package.json
+++ b/package.json
@@ -46,7 +46,7 @@
     "csv": "^1.1.0",
     "debug": "^2.2.0",
     "optimist": "^0.6.1",
-    "sphere-node-sdk": "^1.22.0",
+    "sphere-node-sdk": "^2.0.0",
     "sphere-node-utils": "^0.9.1",
     "tmp": "0.0.33",
     "underscore": "^1.8.3",
@@ -54,7 +54,7 @@
   },
   "devDependencies": {
     "coffee-script": "^1.12.7",
-    "coveralls": "2.11.2",
+    "coveralls": "2.13.2",
     "grunt": "0.4.5",
     "grunt-bump": "0.0.13",
     "grunt-cli": "0.1.13",
```

Reply:
everything else should be reverted since they are not related to the issue context